modules/core/include/opencv2/core.hpp

```diff
@@ -1595,6 +1595,7 @@ have the same size as x.
 @sa cartToPolar, polarToCart, phase, sqrt
 */
 CV_EXPORTS_W void magnitude(InputArray x, InputArray y, OutputArray magnitude);
+CV_EXPORTS_W void magnitude(InputArray xy, OutputArray magnitude);
```

Comment:
There is conflict with Python bindings. Second variant with both variables doesn't not work (would select the first variant - output are optional parameter).

`magnitudeComplex()`?
'c' suffix below has meaning of 'complex' numbers.
But `xy` name has no sense. It is better to use `src` instead.

Reply:
I just followed the same naming convention as in cv::cuda;;magnitude()